diff --git a/node_modules/expo-crypto/build/Crypto.js b/node_modules/expo-crypto/build/Crypto.js
index 21cdd20..efb040b 100644
--- a/node_modules/expo-crypto/build/Crypto.js
+++ b/node_modules/expo-crypto/build/Crypto.js
@@ -2,7 +2,6 @@ import { UnavailabilityError } from 'expo-modules-core';
 import { CryptoDigestAlgorithm, CryptoEncoding } from './Crypto.types';
 import ExpoCrypto from './ExpoCrypto';
 export * from './Crypto.types';
-export * from './aes';
 class CryptoError extends TypeError {
     code = 'ERR_CRYPTO';
     constructor(message) {
